packages/grid/frontend/package.json

```diff
@@ -3,14 +3,11 @@
   "version": "0.8.0-beta.8",
   "private": true,
   "scripts": {
-    "dev": "pnpm install && vite dev --host --port 80",
+    "dev": "vite dev --host --port 80",
```

Comment:
@tcp i think this is needed to ensure that the macOS packages on the outside of the container get switched to linux packages on the inside